src/main/java/io/github/mzmine/datamodel/features/types/FeatureShapeType.java

```diff
@@ -91,7 +90,7 @@ public Node getCellNode(TreeTableCell<ModularFeatureListRow, Boolean> cell,
 
   @Override
   public double getColumnWidth() {
-    return DEFAULT_GRAPHICAL_CELL_WIDTH;
+    return LARGE_GRAPHICAL_CELL_WIDTH;
```

Comment:
why change this now?

Reply:
The chromatograms can be quite wide and its better to start out with a wider column. Better to see if there is a shoulder on a peak or if there are multiple peaks in a chromatogram

Reply:
Making the chromatogram plot wider won't change that, as the visible X axes range is determined by the fwhm of the feature.